go.mod

```diff
@@ -56,7 +60,7 @@ replace (
 	golang.org/x/tools => golang.org/x/tools v0.0.0-20190821162956-65e3620a7ae7 // pinned to release-branch.go1.13
 	k8s.io/api => k8s.io/api v0.18.2
 	k8s.io/apimachinery => k8s.io/apimachinery v0.18.2
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.2
+	k8s.io/cli-runtime => github.com/openshift/kubernetes-cli-runtime v0.0.0-20200507115657-2fb95e953778
```

Comment:
I had to do this in order to get `go mod tidy` to run and pick up the new deps

Reply:
this dominoed down to oc, and re-pinned o/api and o/client-go to weird versions that I had to overwrite there